nisystemlink/clients/alarm/models/_alarm.py

```diff
@@ -1,10 +1,33 @@
 from datetime import datetime
-from enum import Enum
-from typing import Dict, List, Optional
+from enum import Enum, IntEnum
+from typing import Any, Dict, List
 
 from nisystemlink.clients.core._uplink._json_model import JsonModel
 
 
+class AlarmSeverityLevel(IntEnum):
+    """Well-known alarm severity levels.
+
+    The service supports custom severity levels greater than 4, but it is generally discouraged.
+    The SystemLink Alarm UI only has display strings for severity levels in the range [1, 4].
+    """
+
+    CLEAR = -1
+    """Indicates an alarm clearance."""
```

Comment:
I think this wording might be easier to understand:

```suggestion
    """Severity level for cleared alarms."""
```